sdk/search/search-documents/test/public/node/searchClient.spec.ts

```diff
@@ -14,26 +13,55 @@ import {
   KnownQueryLanguage,
   KnownSpeller,
   SearchClient,
+  SearchIndex,
   SearchIndexClient,
   SelectFields,
 } from "../../../src";
 import { SearchFieldArray, SelectArray } from "../../../src/indexModels";
-import { delay, serviceVersions } from "../../../src/serviceUtils";
+import { defaultServiceVersion, delay } from "../../../src/serviceUtils";
 import { COMPRESSION_DISABLED } from "../../compressionDisabled";
 import { Hotel } from "../utils/interfaces";
 import { createClients } from "../utils/recordedClient";
 import { createIndex, createRandomIndexName, populateIndex, WAIT_TIME } from "../utils/setup";
 
-versionsToTest(serviceVersions, {}, (serviceVersion, onVersions) => {
-  onVersions({ minVer: "2023-11-01" }).describe("SearchClient tests", function (this: Suite) {
+describe("SearchClient", function (this: Suite) {
+  this.timeout(99999);
```

Comment:
That's one hell of a timeout 😄 is there any way we can be more conservative or do they really expect to take this long to run?